### ISSUE TITLE
Feature request: Send geo intents to native devices for selecting navigation apps

### ISSUE THREAD

**Issue by leolux** [w3c/web-share#280]
Is it possible to to extend the current web share API by a new feature: Send intents to Android devices for the use case: The user wants to travel to a destination by click on some "start navigation" button and then the web application lets the user decied choose from all naviagtion apps installed on the device, for example Google Maps and Waze. 

Checkout this example how it works already:  https://stackoverflow.com/questions/32437494/is-there-a-way-to-intent-all-navigation-apps-in-phone

Where to suggest a new web intent api that can send geo intents?

**Comment by marcoscaceres** [comment]
Right, this isn't really a "share", unless we were to share location objects (`GeolocationPosition`)... it's more of a navigation.

**Comment by marcoscaceres** [comment]
Closing as "geo" is already in the HTML safe list schemes: 
https://html.spec.whatwg.org/multipage/system-state.html#safelisted-scheme

If it's not supported, it might need to be raised with the implementer (i.e., this is a Chrome bug).
